### PR TITLE
Descriptive error on failed cert generation

### PR DESCRIPTION
**Pull request by bskiba**
#3388 #3379

### CODE REVIEW

File: vertical-pod-autoscaler/pkg/admission-controller/gencerts.sh [hunk 1]
```diff
@@ -38,7 +40,7 @@ EOF
 
 # Create a certificate authority
 openssl genrsa -out ${TMP_DIR}/caKey.pem 2048
-openssl req -x509 -new -nodes -key ${TMP_DIR}/caKey.pem -days 100000 -out ${TMP_DIR}/caCert.pem -subj "/CN=${CN_BASE}_ca" -addext "subjectAltName = DNS:${CN_BASE}_ca"
+openssl req -x509 -new -nodes -key ${TMP_DIR}/caKey.pem -days 100000 -out ${TMP_DIR}/caCert.pem -subj "/CN=${CN_BASE}_ca" -addext "subjectAltName = DNS:${CN_BASE}_ca" || (echo "ERROR: Failed to create CA certificate for self-signing. If the error is \"unknown option -addext\", update your openssl version or deploy VPA from the vpa-release-0.8 branch." && exit 1)
```
Comment (krzysied) [inline vertical-pod-autoscaler/pkg/admission-controller/gencerts.sh:43]: Is there a way to implement it in a different way. More like:
```
command(...)
if status not ok {
  echo error_description
  exit 1
}
```
Having this whole logic in one line is very hard to read and to understand.

Comment (bskiba) [reply vertical-pod-autoscaler/pkg/admission-controller/gencerts.sh]: Done

**Review by mwielgus — APPROVED** [review]
/lgtm
/approve

**Pull request merged**